server/api/users.js

```diff
@@ -88,17 +87,20 @@ router.post('users.s3Upload', auth(), async ctx => {
   ctx.assertPresent(size, 'size is required');
 
   const { user } = ctx.state;
+  const id = uuid.v4();
   const s3Key = uuid.v4();
   const key = `uploads/${user.id}/${s3Key}/${name}`;
   const credential = makeCredential();
   const longDate = format(new Date(), 'YYYYMMDDTHHmmss\\Z');
   const policy = makePolicy(credential, longDate);
   const endpoint = publicS3Endpoint();
   const acl = process.env.AWS_S3_ACL || 'private';
-  const url = acl === 'private' ? proxyS3Url(key) : `${endpoint}/${key}`;
+  const url = `${endpoint}/${key}`;
 
   await Attachment.create({
```

Comment:
`const attachment = await Attachment.create({` above to get the attachment record back with the `id` rather than passing it in below.

Reply:
done